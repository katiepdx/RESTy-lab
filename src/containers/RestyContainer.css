* {
  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
}

body {
  background-color:rgb(134, 65, 65);
  min-width: min-content;
  max-width: 100%;
}

h1 {
  background-color: rgb(223, 189, 189);
  text-align: center;
  padding: 1.2rem;
  border-radius: 1.2rem;
}

.historyAndResponse {
  background-color: rgb(202, 202, 202);
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  column-gap: 2rem;
  padding: 0 1.2rem;
  margin: 1.2rem 0;
  border-radius: 1.2rem;
}
