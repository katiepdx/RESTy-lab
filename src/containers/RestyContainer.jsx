// app logic
import React, { Component } from 'react'

export default class RestyContainer extends Component {
  // state - url, method, click, json
  render() {
    return (
      <>
        <h1>Resty Lab</h1>
        <div>
          Resty Container
        </div>
      </>
    )
  }
}
