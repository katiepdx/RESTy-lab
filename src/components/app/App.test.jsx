import React from 'react';
import { render, cleanup } from '@testing-library/react';
import RestyContainer from '../../containers/RestyContainer';

describe('App component', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(<RestyContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
