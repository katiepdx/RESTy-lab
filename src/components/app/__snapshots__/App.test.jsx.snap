// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<DocumentFragment>
  <h1>
    Resty Lab
  </h1>
  <div>
    Resty Container
  </div>
</DocumentFragment>
`;
